### DISCUSSION TITLE
option to enforce-secure-profile=false popups

### DISCUSSION THREAD

**Discussion by ultraic-space** [PurpurMC/Purpur#1167] [__Dismissed__] [answered]
Not sure what the dev's standpoints are on this but this would be a great feature built-in instead of a separate plugin.
Currently using this, it allows for enforce-secure-profile (enables chat reporting) to be false without sending that popup.
https://github.com/KaspianDev/AntiPopup

**Comment by YouHaveTrouble** [comment, marked as answer]
The popup is there to inform the user that the chat reporting is disabled and messages might be modified by the server, it would be disingenuous to hide that while not having chat reports enabled imho
